src/dra/5087C.c

```diff
@@ -351,13 +351,13 @@ s32 func_800F16D0(void) {
     }
 }
 
-void func_800F1770(s8 arg0[], s32 arg1, s32 arg2, s32 arg3) {
+void func_800F1770(s8 arg0[], s32 arg1, s32 arg2, s32 isEnabled) {
     s32 temp_v0 = (arg1 / 2) + (arg2 * 4);
 
     if (!(arg1 & 1)) {
-        arg0[temp_v0] = (arg0[temp_v0] & 0xF0) + arg3;
+        arg0[temp_v0] = (arg0[temp_v0] & 0xF0) + isEnabled;
     } else {
-        arg0[temp_v0] = (arg0[temp_v0] & 0xF) + (arg3 * 0x10);
+        arg0[temp_v0] = (arg0[temp_v0] & 0xF) + (isEnabled * 0x10);
```

Comment:
If it matches `arg3 << 4` probably makes sense since it's filling out the top nibble.